js/Game.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+class Game extends Drawable {
+  constructor(width, height) {
+    super(); 
+
+    this.width = width;
+    this.height = height;
+
+    this.snake = null;
+    this.fruit = null;
+
+    this.score = null;
+    this.gameOver = false;
+
+    document.addEventListener('keydown', (e) => this.onKeyDown(e));
+  }
+
+  init() {
+    this.snake = this.createSnake();
+    this.fruit = new Fruit(this.getRandomFruitPosition());
+
+    this.score = 0;
+    this.gameOver = false;
+  }
+
+  createSnake() {
+    const x = rand(3, 17);
```

Comment:
What if we set `<canvas id="canvas" width="200" height="200" ...>` in the `index.html`?